csharp_dotnetcore/12.NLP-With-LUIS/LuisBot/BotServices.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights;
+using Microsoft.Bot.Builder.AI.Luis;
+
+namespace Luis_Bot
+{
+    /// <summary>
+    /// Represents the bot's references to external services.
+    ///
+    /// For example, Application Insights and Luis services
```

Comment:
Luis or LUIS? Mostly when we talk about it as a stand-alone word, it's LUIS. I'm not sure what the branding guidelines are on this, but we should be consistent...